### ISSUE TITLE
Add missing telescope keybindings when pressing <leader>

### ISSUE THREAD

**Issue by andradei** [kabinspace/AstroVim#265]
Missing commands:

Any hint that the telescope keybindings exist (`<leader> ff` and others)

![CleanShot 2022-04-08 at 09 23 41](https://user-images.githubusercontent.com/2653546/162473242-e3f6c7ec-04ed-4f4d-a273-38eec07b1d30.png)

**Comment by mehalter** [comment]
Thanks for pointing this missing menu out! Will add it in!

**Comment by mehalter** [comment]
I added the rest of the telescope bindings. Let me know if you find any other discrepancies! Thanks again.

**Comment by andradei** [comment]
It works!
And I was too late, I was looking at https://github.com/kabinspace/AstroVim/wiki/Default-Mappings#telescope-mappings
and it looks like this whole section is missing too
![CleanShot 2022-04-08 at 09 59 27](https://user-images.githubusercontent.com/2653546/162479435-96aeb73d-8b66-404d-ab04-d8fda9613df3.png)

Should I create a new issue?

**Comment by andradei** [comment]
Maybe Load Session and Save Session should not start with `s` because that's specific for `+Search` (registers).

**Comment by mehalter** [comment]
Thanks for this information! :) I think this has exposed a really bad design aspect with how we are configuring which-key. About to open a PR to fix this as well as changing a couple of these mappings to reflect better in which-key

**Comment by mehalter** [comment]
Ok the session bindings and the rest of the dashboard bindings are now set. I went ahead and removed the `<leader>db` binding since `<leader>d` will intercept it anyway. I also moved `<leader>s` to `<leader>S` for Session and made a menu in which-key for it. This PR also extends the mapping and which-key generation so if a plugin is removed in a user config it will not add those entries into which-key
